docs/uma-tokenholders/guidance-on-adding-price-identifiers.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Guidance for the Addition of Price Identifiers
+sidebar_label: Guidance for the Addition of Price Identifiers
+---
+
+Formulating a real-life problem and use case is a good starting point for designing a product to fit market requirements. One can get highly valued feedback by initiating community discussion before the launch of the product. Involve your project community and discuss your idea at UMA Discourse [Ideas Bank](https://discourse.umaproject.org/c/ideas-bank/23) to get started.
+
+This guide covers relevant design considerations and standard practices useful when writing a price identifier UMIP for your product.
+
+## Design Considerations
+
+Even though technically, a price identifier is not tied to any specific application and once approved, can be used by any requester contract - it still might be a good idea to consider the requirements of the application, plan how all the building blocks would fit together and how that should be reflected in the particular price identifier UMIP.
+
+Depending on the type of product one could choose among financial contract templates developed by UMA (e.g. [EMP contracts](/synthetic-tokens/expiring-synthetic-tokens.md) for liquidatable sponsor positions or [LSP contracts](/synthetic-tokens/long-short-pair.md) for un-liquidatable capped-payout applications) or develop Optimistic Oracle integration customized for the particular product. LSP contracts coupled with application-specific financial product libraries as well as Optimistic Oracle integration contracts currently allow for greater customization options that can handle all payout logic; hence the price identifier UMIP can be more lightweight focusing only on how to resolve raw input data (e.g., asset price for call options or TVL or another metric for KPI options). When multiple resolvable input parameters are required (e.g., for expected volatility index calculations), the application logic should still be detailed in the UMIP.
```

Comment:
There are a lot of examples you use throughout the writing and these are great examples but they make the sentences hard to read. 

As a suggestion: It might be better to take out the individual examples and have the paragraph be a description of the heading then as a separate paragraph under the same heading give examples. 

##Heading
Explainer
Example 1
Example 2

I'll highlight the areas I see this.